src/ploomber_core/exceptions.py

```diff
@@ -87,6 +87,8 @@ def _add_community_link(e):
     elif COMMUNITY not in e.args[0]:
         message = e.args[0] + COMMUNITY
         e.args = (message,)
+        if hasattr(e, "message"):
```

Comment:
just add a short comment explaining why we have this edge case

Reply:
[fixed](https://github.com/ploomber/core/pull/81/commits/05685ff1a13f68e62419832249dd806b772260dc)